### ISSUE TITLE
Regular Expression Denial of Service (ReDoS)

### ISSUE THREAD

**Issue by schoenwaldnils** [benmosher/eslint-plugin-import#868]
I updated to v2.3.0 and low snyk.io returns what the issue title says.
Also this:
```
Vulnerable module: ms
Introduced through: eslint-plugin-import@2.3.0 › eslint-module-utils@2.0.0 › debug@2.2.0 › ms@0.7.1
```
More info to this vulnerability: https://snyk.io/vuln/npm:ms:20170412

Is there a quick way to fix this?

**Comment by ljharb** [comment]
Just because there's a vulnerability, does not mean it necessarily affect you. In this case only your own eslint configuration can provide a regular expression, so the risk does not exist.

That said, it'd be ideal to update the dep chain.

**Comment by benmosher** [comment]
fixed in v2.5.0 or so
